src/main/java/com/immunopass/utils/Utils.java

```diff
@@ -0,0 +1,37 @@
+package com.immunopass.utils;
+
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+import org.apache.commons.lang3.RandomStringUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Utils {
```

Comment:
Please rename this class to an appropriate name which tells the functionality.

Reply:
Okay.